example/plot_dataset.py

```diff
@@ -12,25 +13,12 @@
     o = codebox.upload("iris.csv", csv_bytes)
 
     # dataset analysis code
-    code = """
-    import pandas as pd
-    import matplotlib.pyplot as plt
-
-    df = pd.read_csv("iris.csv", header=None)
-    df.columns = ["sepal_length", "sepal_width", "petal_length", "petal_width", "class"]
-
-    # Create a color dictionary for each class label
-    color_dict = {'Iris-setosa': 0, 'Iris-versicolor': 1, 'Iris-virginica': 2}
-
-    # Map the class labels to numbers
-    df['color'] = df['class'].map(color_dict)
-
-    df.plot.scatter(x="sepal_length", y="sepal_width", c="color", colormap="viridis")
-    plt.show()
-    """
+    file_path = Path(__file__).parent / "async_dataset_code.py"
```

Comment:
If you can create a folder "assets" in the example folder and then put booth files in there would be perfect. Also I think they should be just called "dataset_code.py/txt".

Reply:
Sounds good